cloudsql-mysql-plugin/src/e2e-test/java/io/cdap/plugin/CloudMySqlClient.java

```diff
@@ -0,0 +1,190 @@
+package io.cdap.plugin;
+
+import io.cdap.e2e.utils.PluginPropertyUtils;
+import org.junit.Assert;
+
+import java.sql.*;
+import java.util.Date;
+import java.util.GregorianCalendar;
+import java.util.TimeZone;
+
+public class CloudMySqlClient {
+
+    private static final String database = PluginPropertyUtils.pluginProp("DatabaseName");
+    private static final String connectionName = PluginPropertyUtils.pluginProp("ConnectionName");
+
+    public static void main(String[] args) throws SQLException, ClassNotFoundException {
```

Comment:
Remove the main method from here we don't need this in production code.

Reply:
resolved